### DISCUSSION TITLE
Inadyn Debian Not Updating Unless Run Manually

### DISCUSSION THREAD

**Discussion by moosefuel** [troglobit/inadyn#432] [General]
Hi again, 

I've worked through getting inadyn to work well, but I still have to run it each time from the command line. It doesn't update in the background when I unplug and re-plug my router to get a new IP. I checked my console to see if it was running as a service, and it says:

<img width="574" alt="Screenshot 2023-07-19 at 08 35 17" src="https://github.com/troglobit/inadyn/assets/111374276/27ae5928-f616-460b-beed-13ce3447f3a8">

So my question is, should this /etc/default/inadyn be enabled? The file is a text file which gives me the option to run it as a daemon and choose which user to run as, but if I check my services I see there is something called inadyn already running.

**Comment by troglobit** [comment]
I'm sorry, but this is a packaging/integration issue that I have no control over.  You need to report it to or ask the system/distribution that packages and uses /etc/default/inadyn how to get it going.

**Reply by moosefuel** [reply]
Understood. Thank you.

For reference for other Debian users, I did edit /etc/default/inadyn to allow it to run and it worked, I was able to unplug, replug, and have *most* of my domains update with the new address. Still troubleshooting the ones that didn't. 

*EDIT* now that I have waited a few more minutes, all IP addresses have updated after a failure. This is working perfectly now, and I am very happy. Thank you for your help in general.
